install.sh

```diff
@@ -119,18 +121,42 @@ install_packages( )
     library_dependencies_centos+=( "" ) # how to install cuda on centos?
     library_dependencies_fedora+=( "" ) # how to install cuda on fedora?
   elif [[ "${build_hip_clang}" == false ]]; then
-    library_dependencies_ubuntu+=( "rocm-dev" "rocblas" )
-    library_dependencies_centos+=( "rocm-dev" "rocblas" )
-    library_dependencies_fedora+=( "rocm-dev" "rocblas" )
-    library_dependencies_sles+=( "rocm-dev" "rocblas" )
+    # Custom rocm-dev installation
+    if [[ -z ${custom_rocm_dev+foo} ]]; then
+      # Install base rocm-dev package unless -v/--rocm-dev flag is passed
+      library_dependencies_ubuntu+=( "rocm-dev" )
+      library_dependencies_centos+=( "rocm-dev" )
+      library_dependencies_fedora+=( "rocm-dev" )
+      library_dependencies_sles+=( "rocm-dev" )
+    else
+      # Install rocm-specific rocm-dev package
+      library_dependencies_ubuntu+=( "${custom_rocm_dev}" )
+      library_dependencies_centos+=( "${custom_rocm_dev}" )
+      library_dependencies_fedora+=( "${custom_rocm_dev}" )
+      library_dependencies_sles+=( "${custom_rocm_dev}" )
+    fi
+
+    # Custom rocblas installation
+    if [[ -z ${custom_rocblas+foo} ]]; then
+      # Install base rocblas package unless -b/--rocblas flag is passed
+      library_dependencies_ubuntu+=( "rocblas" )
+      library_dependencies_centos+=( "rocblas" )
+      library_dependencies_fedora+=( "rocblas" )
+      library_dependencies_sles+=( "rocblas" )
+    else
+      # Install rocm-specific rocblas package
+      library_dependencies_ubuntu+=( "${custom_rocblas}" )
+      library_dependencies_centos+=( "${custom_rocblas}" )
+      library_dependencies_fedora+=( "${custom_rocblas}" )
+      library_dependencies_sles+=( "${custom_rocblas}" )
+    fi
+  fi
 
-    if [[ "${build_solver}" == true ]]; then
+  if [[ "${build_solver}" == true ]]; then
       library_dependencies_ubuntu+=( "rocsolver" )
       library_dependencies_centos+=( "rocsolver" )
       library_dependencies_fedora+=( "rocsolver" )
       library_dependencies_sles+=( "rocsolver" )
```

Comment:
Now I am not sure why these other dependencies aren't set for hip-clang, but if that is true solver used to be also excluded for hip-clang (was inside the if ).   Sorry but this may be question for CI who are on holiday.

Reply:
That's true, looks like this section was inside excluded from hip-clang. Hm, I'm not sure of the implications of this either.
I just built using hip-clang on my local machine and it seemed to be ok.

Reply:
We can ask Akila next week

Reply:
Sounds good. Will merge into develop for now.